xcode/UnusedConfig.yml

```diff
@@ -0,0 +1,5 @@
+--ignored-resources:
```

Comment:
`--` is legal in yaml?

Reply:
do like swiftlint:
https://gist.github.com/tweetjay/1b7f00b6f312a5a2cf7b5676420b9b0c#file-swiftlint-yml-L3